receive.js

```diff
@@ -6,6 +6,10 @@ exports.process = function (msg, conf) {
     console.log("Received new message with id", msgId);
     console.log(msg);
 
+    if (msg.query && msg.body) {
```

Comment:
What if we have query string params on a GET request?

Reply:
@jhorbulyk you mean something like:

```
https://in.elastic.io/hooks/123456789?query=foo
```
?

Reply:
Correct.

Reply:
they result in `msg.body` anyway

Reply:
When would `body` not be defined?  If one does a POST or similar with no body, is there something to attach the query string params to?

Reply:
`msg.body` is always defined.